tests/features/cookie-consent-kit.feature

```diff
@@ -0,0 +1,12 @@
+@api
+Feature: Cookie consent kit.
+  In order to show usage of cookie consent kit on the website
+  As a anonymous user
+  I want to be able to see usage of CCK for remote videos.
```

Comment:
I want to be able to see the usage of CCK for remote videos.

Reply:
fixed